usaspending_api/awards/migrations/0110_financialaccountsbyawards_faba_object_sumission_idx.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.19 on 2025-04-10 18:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("awards", "0109_delete_old_park_columns_faba"),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name="financialaccountsbyawards",
+            index=models.Index(
+                condition=models.Q(("transaction_obligated_amount", False)),
```

Comment:
```suggestion
                condition=models.Q(("transaction_obligated_amount__isnull", False)),
```
https://docs.djangoproject.com/en/5.2/ref/models/querysets/#isnull